META-INF/com/google/android/update-binary

```diff
@@ -108,6 +108,14 @@ else
 error_mounting "$SYSTEM_MNT"
 fi
 
+if [[ `ls ${SYSTEM_MNT}/system/bin/ih8sn 2> /dev/null` ]]; then
+  ui_print "Removing existing ih8sn files"
+  find ${SYSTEM_MNT}/system | grep ih8sn | xargs rm
```

Comment:
`find ${SYSTEM_MNT}/system -name *ih8sn* -delete`

Reply:
similar suggestion to other places with `find | grep | xargs`

Reply:
done